.goreleaser.yaml

```diff
@@ -249,32 +247,32 @@ winget:
 after:
   hooks:
     - cmd: brew bump-cask-pr rio --version {{ .Version }}
-      if: "{{ not .IsNightly }}"
+      if: "{{ .IsRelease }}"
```

Comment:
this takes into account --snapshot as well - before these hooks would run if --snapshot was used